checkov/common/bridgecrew/run_metadata/registry.py

```diff
@@ -0,0 +1,16 @@
+
+
+class RunMetadataExtractorsRegistry():
+    def __init__(self):
+        self.extractors = set()
+
+    def register(self, extractor) -> None:
+        self.extractors.add(extractor)
+
+    def get_extractor(self):
+        for extractor in self.extractors:
+            if extractor.is_current_ci:
```

Comment:
```suggestion
            if extractor.is_current_ci():
```
does it wok without the round brackets?